edgelet/edgelet-docker/src/settings.rs

```diff
@@ -450,6 +478,8 @@ mod tests {
         "test/windows/bad_sample_settings.dyn.repro.yaml";
     #[cfg(windows)]
     static GOOD_SETTINGS_TLS: &str = "test/windows/sample_settings.tls.yaml";
+    #[cfg(windows)]
+    static GOOD_SETTINGS_IMAGE_RESOLVE: &str = "test/linux/sample_settings_image_resolve.yaml";
```

Comment:
Are you using the same .yaml from linux for windows?

Reply:
Yes, there is not difference in the tested code. It will just duplicate the yaml

Reply:
I'll add it to be consistent with other tests

Reply:
Fixed it